frontend/src/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>WebMUD3 UNItopia</title>
-    <base href="/" />
+    <base href="./">
```

Comment:
Hast du das bereits in den verschiedenen Konstellationen ausprobiert?

- https://www.unitopia.de/webmud3/
- https://www.unitopia.de/webmud3test/orbit
- https://seifenblase.de/webmud3/

waren die Ziele, die du damals angegeben hast

Reply:
das passt für webmud3test soweit
ist aber noch ein sehr manueller Prozess, das Dockerfile bietet derzeit nur ein bash Skript an und startet noch nicht das webmud3 automatisch

Reply:
Wurde bereits mit https://github.com/mystiker/webmud3/pull/50 gemerged.